packages/components/src/components/primitives/Checkbox/types.ts

```diff
@@ -0,0 +1,8 @@
+import { CheckboxProps as ChakraCheckboxProps } from '@chakra-ui/react';
+
+export const CheckboxSizes = ['sm', 'md', 'lg'] as const;
```

Comment:
sizes are defined as `s`, `m` and `l` in figma

Reply:
done!